scripts/gcplog/main.tf

```diff
@@ -0,0 +1,40 @@
+terraform {
+  required_providers {
+    google = {
+      source = "hashicorp/google"
+      version = "3.5.0"
+    }
+  }
+}
+
+variable "credentials_file" {}
+variable "zone" {}
+variable "region" {}
+variable "project" {}
+variable "logname" {
+  default = "cloud-logs"
+}
+
+provider "google" {
+  credentials = file(var.credentials_file)
+  project = var.project
+  zone = var.zone
+  region= var.region
+
+}
+
+resource "google_pubsub_topic" "cloud-logs" {
+  name= var.logname
+}
+
+resource "google_logging_project_sink" "cloud-logs" {
+  name = var.logname
+  destination = "pubsub.googleapis.com/projects/personal-226821/topics/${var.logname}"
+  filter = "resource.type = gcs_bucket AND severity >= WARNING"
```

Comment:
currently, filter is hardcoded. I will add it as variable in future.

Reply:
You should add to the doc a small section about this filter, the langage , what it does, and let's suggest a sane default ? What do you think would be a good default for someone that wants to get visibility into his cloud ?

Reply:
This might be a sane default. Perhaps we can link to GCP's filter documentation in the README, but I don't think we need to add much tooling here to change it -- this can serve as a template.

Reply:
I agree with filter doc and sane defaults.
1. Added some explanation on the README with doc link
2. I think sane default would be some cloud resource most of the customer be having. `httpLoadBalancer`? so I added that. wdyt?